src/app/state/rest/hooks/utils/utils.ts

```diff
@@ -1,14 +1,23 @@
 import { useCallback } from "react"
+import { navigate } from "@reach/router"
 import { AxiosError } from "axios"
 import qs from "qs"
 import slashSandwich from "slash-sandwich"
 
-import { getToken } from "app/state/auth/tokenStore"
+import {clearToken, getToken} from "app/state/auth/tokenStore"
+import to from "app/features/shared/routing/to";
 
 /**
  * Default error handler:
  */
-export const useErrorHandler = () => useCallback((error: AxiosError) => alert(error.message), [])
+export const useErrorHandler = () => useCallback(async (error: AxiosError) => {
+  if (error?.response?.status === 401) {
```

Comment:
`error` can not be undefined. Or it should also be an optional here `alert(error.message)`.

Reply:
`error` could be any type of error, not just AxiosError.
Fixed typings.